frontend/src/pages/AddForm/AddForm.tsx

```diff
@@ -53,7 +54,11 @@ const AddForm = (): JSX.Element => {
 
   const onCancel = () => history.replace('/admin');
   const onSubmit = async (values: any) => {
-    await createUser(values);
+    const result = await createUser(values);
+    if (result.error && result.reason.toLowerCase() === 'this email already exists') {
```

Comment:
This solution fits only for incorrect email. What about we will face issue with other field? Like boss (in case we are choosing boss, but in the meantime someone remove it from DB) In this case we should see an error under boss field. Maybe we could make this error handler more generic?